docs/install.md

```diff
@@ -236,6 +226,13 @@ alternative](https://hub.docker.com/r/linuxserver/code-server).
 
 You can install code-server via [Helm](https://github.com/cdr/code-server/blob/main/ci/helm-chart/README.md).
 
+## Windows
+
+We currently do not publish windows releases (see [#1397](https://github.com/cdr/code-server/issues/1397)). We recommend installing code-server onto Raspberry Pi with [`yarn` or
```

Comment:
the first sentence talks about Windows, the second sentence talks about Raspberry Pi? Also, I think Windows should be capitalized

Reply:
thanks for the review, good catch.